docs/source/getting-started/registering-first-model/step3-load-model.rst

```diff
@@ -0,0 +1,84 @@
+Load a Registered Model
+=======================
+
+To perform inference on a registered model version, we need to load it into memory. There are many 
+ways to find our model version, but the best method differs depending on the information you have
+available. However, in the spirit of a quickstart, here is the method that requires the least amount
+of information.
+
+.. code-block:: python
+
+    import mlflow.sklearn
+    from sklearn.datasets import make_regression
+
+    model_name = "sk-learn-random-forest-reg-model"
+    model_version = "latest"
+
+    # Load the model from the Model Registry
+    model_uri = f"models:/{model_name}/{model_version}"
+    model = mlflow.sklearn.load_model(model_uri)
+
+    # Generate a new dataset for prediction and predict
+    X_new, _ = make_regression(n_features=4, n_informative=2, random_state=0, shuffle=False)
+    y_pred_new = model.predict(X_new)
+
+    print(y_pred_new)
+
+
+Example 1: Load via Static Information
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+There are many other ways to construct a model URI. Here are some common examples that leverage 
+information created at registration time. 
+
+1. **Absolute local path**: ``mlflow.sklearn.load_model("/Users/me/path/to/local/model")``
+2. **Relative local path**: ``mlflow.sklearn.load_model("relative/path/to/local/model")``
+3. **Run id**: ``mlflow.sklearn.load_model(f"runs:/{mlflow_run_id}/{run_relative_path_to_model}")``
+4. **Model name + version**: ``mlflow.sklearn.load_model(f"models:/{model_name}/{model_version}")``
```

Comment:
Well, only this model URI is specific to the model registry. The other URIs are relevant to regular MLflow experimentation. Maybe example 1 could just be the model name and version number URI?